core/src/main/java/com/nerds/gamejam/ecs/component/TextureReferenceComponent.java

```diff
@@ -0,0 +1,56 @@
+package com.nerds.gamejam.ecs.component;
+
+import com.artemis.Component;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.Texture;
+import com.badlogic.gdx.graphics.Texture.TextureWrap;
+
+import java.util.List;
+
+public class TextureReferenceComponent extends Component {
```

Comment:
Hmm for clarity I'd have left the single/composite idea in play as this is a bit funky having a TextureReferenceComponent use a list of references, if you want a single one you need to create a list which seems unnecessary

Reply:
Added a constructor for a single texture reference, having the one component just seems to make the rendering code a bit easier to understand. Less if this then this else this else this stuff going on